spec/requests/admin_claim_checks_spec.rb

```diff
@@ -8,33 +8,31 @@
       follow_redirect!
     end
 
-    describe "claim_approvals#create" do
+    describe "claim_checks#create" do
       let(:claim) { create(:claim, :submitted) }
 
       it "approves a claim" do
         freeze_time do
-          post admin_claim_approvals_path(claim_id: claim.id)
+          post admin_claim_checks_path(claim_id: claim.id, result: "approved")
 
           follow_redirect!
 
           expect(response.body).to include("Claim has been approved successfully")
 
-          claim.reload
-
-          expect(claim.approved_at).to eq(Time.zone.now)
-          expect(claim.approved_by).to eq("123")
+          expect(claim.check.checked_by).to eq("123")
+          expect(claim.check.result).to eq("approved")
```

Comment:
Shouldn't we be checking the creation date, too?

Reply:
yep, will add this.

Reply:
As above, I don't think we need to explicitly test this as it's baked-in ActiveRecord behaviour.